### PR TITLE
docs(zh): add Chinese docs

### PR DESCRIPTION
**Pull request by KimYangOfCat**
Hello, I want to add chinese docs for pinia. I had added `zh` folder, and completed the config for  localisation. 
But I don't know if the official team is willing to add Chinese documentation，so I created this PR for comments. 
I will complete the translation of the document in the next few days, if the official wishes.

### CODE REVIEW

**Comment by KimYangOfCat** [comment]
Hi, everyone, I am so happy to tell you that I have finished the chinese docs translation. If you are good to Chinese , I am willing to get your suggestion of review.
Now, you can click [here](https://deploy-preview-1193--pinia-official.netlify.app/) to preview.

**Comment by PatrickChen928** [comment]
image
这里翻译成【商店】？我觉得应该是直接用store吧？

**Comment by murongg** [comment]
> image
> 
> 这里翻译成【商店】？我觉得应该是直接用store吧？

这个可能是作者机器翻译的😄

**Comment by PatrickChen928** [comment]
> > image
> > 这里翻译成【商店】？我觉得应该是直接用store吧？
> 
> 这个可能是作者机器翻译的😄

估计是的。可能遗漏了吧😄

**Comment by PatrickChen928** [comment]
image

**Comment by KimYangOfCat** [comment]
> 这个可能是作者机器翻译的😄

为了提升速度，翻译过程中确实有部分借助机翻，但都经过一次人工校对，这里应该是我疏忽了，多谢指出，我立马修改。

下次提意见的时候，如果可以的话，请按[标准的 review 方式](https://docs.github.com/cn/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)操作吧。这样可以加快 BUG 修改，你也可以成为本 PR 的贡献者，当本 PR 被 merge 之后你也可以成为仓库贡献者。目前这样的截图方式并不是很方便。

**Comment by PatrickChen928** [comment]
> > 这个可能是作者机器翻译的😄
> 
> 为了提升速度，翻译过程中确实有部分借助机翻，但都经过一次人工校对，这里应该是我疏忽了，多谢指出，我立马修改。
> 
> 下次提意见的时候，如果可以的话，请按[标准的 review 方式](https://docs.github.com/cn/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)操作吧。这样可以加快 BUG 修改，你也可以成为本 PR 的贡献者，当本 PR 被 merge 之后你也可以成为仓库贡献者。目前这样的截图方式并不是很方便。

ok. 我的疏忽。大致翻了下看到的问题，就直接评论了。下次我注意～

**Review by PatrickChen928 — APPROVED** [review]

File: packages/docs/zh/cookbook/testing.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+```ts
+import { defineStore } from 'pinia'
+
+const useCounterStore = defineStore('counter', {
+  state: () => ({ n: 0 }),
+  // ...
+})
+```
+
+Since the store is named _"counter"_, you need to add a matching object to `initialState`:
```
Comment (aimer-fan) [inline packages/docs/zh/cookbook/testing.md]: 由于store的名字叫counter，所以你需要传递相应的对象给`initialState`:

File: packages/docs/zh/cookbook/testing.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+```js
+import sinon from 'sinon'
+
+createTestingPinia({
+  createSpy: sinon.spy, // use sinon's spy to wrap actions
+})
+```
+
+你可以在[测试包的测试]（https://github.com/vuejs/pinia/blob/v2/packages/testing/src/testing.spec.ts）中找到更多的例子。
```
Comment (aimer-fan) [inline packages/docs/zh/cookbook/testing.md]: 你可以在[测试包的测试](https://github.com/vuejs/pinia/blob/v2/packages/testing/src/testing.spec.ts)中找到更多的例子。

File: packages/docs/zh/cookbook/composing-stores.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+      return `Hi ${user.name}, you have ${state.list.length} items in your cart. It costs ${state.price}.`
+    },
+  },
+})
+```
+
+## 共享 Actions {#shared-actions}
+
+The same applies to _actions_:
```
Comment (aimer-fan) [inline packages/docs/zh/cookbook/composing-stores.md]: 同样适用于_actions_:

**Review by aimer-fan — CHANGES_REQUESTED** [review]
翻译遗漏和格式错误

**Comment by KimYangOfCat** [comment]
@Aimer-Fan  Thanks for your review, I have fixed them. If no other suggestions, please approve this PR, thanks again!

**Review by aimer-fan — APPROVED** [review]

**Review by arnonrdp — APPROVED** [review]
This is a big step towards opening up more languages to documentation.

Please merge as soon as possible.

**Comment by KimYangOfCat** [comment]
@posva hi, there are more than two people that has approved this PR , so can it be merged as soon as possible?

File: packages/docs/zh/introduction.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
+# 简介 {#introduction}
+
+<VueSchoolLink
+  href="https://vueschool.io/lessons/introduction-to-pinia"
+  title="Get started with Pinia"
+/>
+
+Pinia [最初](https://github.com/vuejs/pinia/commit/06aeef54e2cad66696063c62829dac74e15fd19e)是在 2019 年 11 月左右作为一个使用 [Composition API](https://github.com/vuejs/composition-api) 的实验进行设计的。从那时起，同时支持 Vue2 和 Vue3 以及不强制要求使用 composition API 的初始原则保留至今。除了**安装**和**SSR**之外，支持 Vue3 和 Vue2 的 API 都是相同的。虽然这些文档主要是面向 Vue3，但在必要时会标注出 Vue2 的内容，因此 Vue2 和 Vue3 的用户都可以阅读本文档。
```
Comment (awxiaoxian2020) [inline packages/docs/zh/introduction.md]:
```suggestion
Pinia [最初](https://github.com/vuejs/pinia/commit/06aeef54e2cad66696063c62829dac74e15fd19e)是在 2019 年 11 月左右作为一个使用 [组合式 API](https://github.com/vuejs/composition-api) 的实验进行设计的。从那时起，同时支持 Vue2 和 Vue3 以及不强制要求使用组合式 API 的初始原则保留至今。除了**安装**和 **SSR** 之外，支持 Vue3 和 Vue2 的 API 都是相同的。虽然这些文档主要是面向 Vue3，但在必要时会标注出 Vue2 的内容，因此 Vue2 和 Vue3 的用户都可以阅读本文档。
```
建议参照 Vue 官方文档翻译成对应中文。

File: packages/docs/zh/introduction.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+
+## 为什么要使用 Pinia？{#why-should-i-use-pinia}
+
+Pinia 是专属 Vue 的 store 库，它允许你跨组件/页面共享状态。如果你熟悉 Composition API 的话，你可能会认为你可以通过一行简单的 `export const state = reactive({})`来共享一个全局状态。对于单页应用程序来说确实是这样的，但如果在服务器端渲染，这可能会使您的应用程序暴露出一些安全漏洞。 不过若是使用 Pinia，即使在小型单页应用程序中，你也可以获得很多好处：
```
Comment (awxiaoxian2020) [inline packages/docs/zh/introduction.md]:
```suggestion
Pinia 是专属 Vue 的状态管理库，它允许你跨组件/页面共享状态。如果你熟悉组合式 API 的话，你可能会认为你可以通过一行简单的 `export const state = reactive({})`来共享一个全局状态。对于单页应用程序来说确实是这样的，但如果在服务器端渲染，这可能会使您的应用程序暴露出一些安全漏洞。 不过若是使用 Pinia，即使在小型单页应用程序中，你也可以获得很多好处：
```
同上。

**Review by awxiaoxian2020 — CHANGES_REQUESTED** [review]

File: packages/docs/zh/introduction.md [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+    ...mapActions(useCounterStore, ['increment']),
+  },
+}
+```
+
+你将会在核心概念部分了解到更多关于 _map helper_ 的信息。
+
+## 为什么命名为 _Pinia_？{#Why-pinia}
+
+Pinia (发音为 `/piːnjʌ/`，类似英文中的 “peenya”) 是最接近有效包名 piña（西班牙语中的 _pineapple_）的词。 菠萝实际上是一组各自独立的花朵，它们结合在一起，由此形成一个多重的水果。 与 Store 类似，每一个都是独立诞生的，但最终它们都是相互联系的。 它也是一种原产于南美洲的美味热带水果。
```
Comment (awxiaoxian2020) [inline packages/docs/zh/introduction.md]:
```suggestion
Pinia (发音为 `/piːnjʌ/`，类似英文中的 “peenya”) 是最接近有效包名 piña（西班牙语中的 _pineapple_，即“菠萝”）的词。 菠萝花实际上是一组各自独立的花朵，它们结合在一起，由此形成一个多重的水果。 与 Store 类似，每一个都是独立诞生的，但最终它们都是相互联系的。 它（菠萝）也是一种原产于南美洲的美味热带水果。
```

File: packages/docs/zh/introduction.md [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+
+## 与 Vuex 对比{#comparison-with-vuex}
+
+Pinia 最初是为了探索 Vuex 的下一次迭代会是什么样子，结合了 Vuex 5 核心团队讨论中的许多想法。最终，我们意识到 Pinia 已经实现了我们在 Vuex 5 中想要的大部分功能，并决定将其作为新的推荐来代替 Vuex。
+
+与 Vuex 相比，Pinia 不仅提供了一个更简单的 API，也提供了Composition-API 风格的 API，最重要的是，与 TypeScript 一起使用时有坚实的类型推断支持。
+
+### RFCs
+
+最初，Pinia 没有经过任何 RFC。我根据自己开发应用程序的经验，同时阅读其他人的代码，为使用 Pinia 的客户工作，以及在 Discord 上回答问题，测试了一些想法。
```
Comment (awxiaoxian2020) [inline packages/docs/zh/introduction.md]:
```suggestion
最初，Pinia 没有经过任何 RFC 的流程。我根据自己开发应用程序的经验，同时阅读其他人的代码，为使用 Pinia 的客户工作，以及在 Discord 上回答问题，验证了一些想法。
```

File: packages/docs/zh/introduction.md [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+这些使我能够产出了这样一个可行的解决方案，并适应各种情况和应用规模。我曾经常发表文章，并在保持其核心 API 不变的情况下不断优化本库。
+
+现在 Pinea 已经成为默认的状态管理解决方案，它和 Vue 生态系统中的其他核心库一样，都要经过 RFC 流程，其 API 已经进入稳定状态。
```
Comment (awxiaoxian2020) [inline packages/docs/zh/introduction.md]:
```suggestion
现在 Pinia 已经成为默认的状态管理解决方案，它和 Vue 生态系统中的其他核心库一样，都要经过 RFC 流程，其 API 已经变得稳定。
```

File: packages/docs/zh/introduction.md [hunk 9]
```diff
@@ -0,0 +1,192 @@
...
+
+### 与 Vuex 3.x/4.x 对比{#comparison-with-vuex-3-x-4-x}
+
+> Vuex 3.x 是适配 Vue2 的 Vuex，而 Vuex 4.x 是适配 Vue3 的。
+
+Pinia API 与 Vuex ≤4 有很大不同，即：
+
+- _mutation_ 不再存在。它们经常被认为是**极其冗长的**。它们最初带来了 devtools 的集成，但这已不再是一个问题了。
```
Comment (awxiaoxian2020) [inline packages/docs/zh/introduction.md]:
```suggestion
- _mutation_ 不再存在。它们经常被认为是**极其冗长的**。它们最初带来了 devtools 的集成方案，但这已不再是一个问题了。
```

**Review by awxiaoxian2020 — CHANGES_REQUESTED** [review]

File: packages/docs/zh/getting-started.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+
+Store（如 Pinia）是一个保存状态和业务逻辑的实体，它并不与你的组件树绑定。换句话说，**它承载着全局状态**。它有点像一个永远存在的组件，每个组件都可以读取和写入它。它有**三个概念**，[state](./core-concepts/state.md)、[getters](./core-concepts/getters.md) 和 [actions](./core-concepts/actions.md)，我们可以假设这些概念相当于组件中的 `data`、 `computed` 和 `methods`。
+
+## 应该在什么时候使用 Store?{#when-should-i-use-a-store}
+
+一个 Store 应该包含可以在整个应用程序中访问的数据。这包括在许多地方使用的数据，例如显示在导航栏中的用户信息，以及需要通过页面保存的数据，例如一个非常复杂的多步骤表单。
+
+另一方面，你应该避免在 Store 中包括那些可以在组件中托管的本地数据，例如，一个元素在页面中的可见性。
+
+并非所有的应用程序都需要访问全局状态，但如果你的应用程序需要一个全局状态，Pinia 将使你的生活更轻松。
```
Comment (awxiaoxian2020) [inline packages/docs/zh/getting-started.md]:
```suggestion
并非所有的应用程序都需要访问全局状态，但如果你的应用程序需要一个全局状态，Pinia 将使你的开发更轻松。
```
感觉英文使用 life 没有问题，但是中文使用“生活”就有点奇怪。

File: packages/docs/zh/ssr/index.md [hunk 11]
```diff
@@ -0,0 +1,111 @@
+# 服务端渲染 (SSR){#server-side-rendering-ssr}
+
+:::tip
+如果你使用的是 **Nuxt.js，**你需要阅读的是[**这些说明文档**](./nuxt.md)。
```
Comment (awxiaoxian2020) [inline packages/docs/zh/ssr/index.md]:
```suggestion
如果你使用的是 **Nuxt.js**，你需要阅读的是[**这些说明文档**](./nuxt.md)。
```

**Review by awxiaoxian2020 — CHANGES_REQUESTED** [review]

**Comment by awxiaoxian2020** [comment]
@KimYangOfCat Could you please check other Composition API translation, and replace the `Store` to "状态管理" when neccessary ( e.g. in index title description) if you have spare time?

**Comment by KimYangOfCat** [comment]
@awxiaoxian2020  thanks for your reminder. this PR has some conflicts due to update now. I will update my branch and translate new content recently, and previous tanslations will be checked again.

**Comment by awxiaoxian2020** [comment]
And the CI seems failed. Do we need to fix it?

**Comment by KimYangOfCat** [comment]
yes, but now there's no rush, I need to finish translating the update first. Then I will fix CI

File: packages/docs/zh/introduction.md [hunk 12]
```diff
@@ -0,0 +1,192 @@
+# 简介 {#introduction}
+
+<VueSchoolLink
+  href="https://vueschool.io/lessons/introduction-to-pinia"
+  title="Get started with Pinia"
+/>
+
+Pinia [起始](https://github.com/vuejs/pinia/commit/06aeef54e2cad66696063c62829dac74e15fd19e)于 2019 年 11 月左右的一场实验，该实验的目的是设计一个拥有 [组合式 API](https://github.com/vuejs/composition-api) 的 Store。从那时起，同时支持 Vue2 和 Vue3 以及不强制要求使用组合式 API 的初心被保留至今。除了**安装**和 **SSR** 等章节之外，其余章节内容的 API 均支持 Vue3 和 Vue2。虽然本文档主要是面向 Vue3，但在必要时会标注出 Vue2 的内容，因此 Vue2 和 Vue3 的用户都可以阅读本文档。
```
Comment (awxiaoxian2020) [inline packages/docs/zh/introduction.md]:
```suggestion
Pinia [起始](https://github.com/vuejs/pinia/commit/06aeef54e2cad66696063c62829dac74e15fd19e)于 2019 年 11 月左右的一次实验，该实验的目的是设计一个拥有 [组合式 API](https://github.com/vuejs/composition-api) 的 Vue 状态管理库。从那时起，我们就倾向于同时支持 Vue2 和 Vue3，并且不强制要求使用组合式 API，我们的初心至今没有改变。除了**安装**和 **SSR** 等章节之外，其余章节内容的 API 均支持 Vue3 和 Vue2。虽然本文档主要是面向 Vue3 的用户，但在必要时会标注出 Vue2 的内容，因此 Vue2 和 Vue3 的用户都可以阅读本文档。
```

File: packages/docs/zh/introduction.md [hunk 13]
```diff
@@ -0,0 +1,192 @@
...
+  - Store 可呈现于使用到它们的组件中
+  - Time travel 让调试更容易
+- 热更新
+  - 不必重载页面即可修改 Store
+  - 开发时可保持当前的 State
+- 插件：可通过插件扩展 Pinia 功能
+- 为 JS 开发者提供适当的 TypeScript 支持以及 **autocompletion** 功能。
+- 支持服务端渲染
+
+## 基础示例{#basic-example}
```
Comment (awxiaoxian2020) [inline packages/docs/zh/introduction.md]:
```suggestion
## 基础示例 {#basic-example}
```

File: packages/docs/zh/introduction.md [hunk 14]
```diff
@@ -0,0 +1,192 @@
...
+}
+```
+
+你将会在核心概念部分了解到更多关于每个 _map helper_ 的信息。
+
+## 为什么命名为 _Pinia_？{#Why-pinia}
+
+Pinia (发音为 `/piːnjʌ/`，类似英文中的 “peenya”) 是最接近有效包名 piña（西班牙语中的 _pineapple_，即“菠萝”）的词。 菠萝花实际上是一组各自独立的花朵，它们结合在一起，由此形成一个多重的水果。 与 Store 类似，每一个都是独立诞生的，但最终它们都是相互联系的。 它（菠萝）也是一种原产于南美洲的美味热带水果。
+
+## 更真实的示例{#a-more-realistic-example}
```
Comment (awxiaoxian2020) [inline packages/docs/zh/introduction.md]:
```suggestion
## 更真实的示例 {#a-more-realistic-example}
```

File: packages/docs/zh/introduction.md [hunk 15]
```diff
@@ -0,0 +1,192 @@
...
+
+这是一个更完整的你将会使用的 Pinia API 示例，即使在 JavaScript 中也具有类型提示。对于某些人来说，可能足以在不进一步阅读的情况下直接开始阅读本节内容，但我们仍然建议你继续阅读文档的其余部分，甚至跳过此示例，在阅读完所有**核心概念**之后再返回。
```
Comment (awxiaoxian2020) [inline packages/docs/zh/introduction.md]:
```suggestion
这是一个更完整的 Pinia API 示例，在 JavaScript 中也使用了类型提示。对于某些人来说，可能足以在不进一步阅读的情况下直接开始阅读本节内容，但我们仍然建议你继续阅读文档的其余部分，甚至跳过此示例，在阅读完所有**核心概念**之后再回来。
```

File: packages/docs/zh/introduction.md [hunk 16]
```diff
@@ -0,0 +1,192 @@
...
+
+## 对比 Vuex {#comparison-with-vuex}
+
+Pinia 起源于一场探索 Vuex 下一个迭代的实验，所以其结合了 Vuex 5 核心团队讨论中的许多想法。最终，我们意识到 Pinia 已经实现了我们在 Vuex 5 中想要的大部分功能，所以决定将其作为新的推荐方案来代替 Vuex。
+
+与 Vuex 相比，Pinia 不仅提供了一个更简单的 API，也提供了符合组合式 API 风格的 API，最重要的是，搭配 TypeScript 一起使用时有非常可靠的类型推断支持。
+
+### RFCs
+
+最初，Pinia 没有经过任何 RFC 的流程。我基于自己开发应用程序的经验，同时通过阅读其他人的代码，为使用 Pinia 的客户工作，以及在 Discord 上回答问题等方式验证了一些想法。
```
Comment (awxiaoxian2020) [inline packages/docs/zh/introduction.md]:
```suggestion
最初，Pinia 没有经过任何 RFC 的流程。我基于自己开发应用程序的经验，同时通过阅读其他人的代码，为使用 Pinia 的用户工作，以及在 Discord 上回答问题等方式验证了一些想法。
```

File: packages/docs/zh/introduction.md [hunk 17]
```diff
@@ -0,0 +1,192 @@
...
+这些使我能够产出了这样一个可行的解决方案，并适应各种情况和应用规模。我一直在保持其核心 API 不变的情况下发布新版本同时不断优化本库。
```
Comment (awxiaoxian2020) [inline packages/docs/zh/introduction.md]:
```suggestion
这些使我产出了这样一个可行的解决方案，并适应各种场景和应用规模。我一直在保持其核心 API 不变的情况下发布新版本，同时不断优化本库。
```

File: packages/docs/zh/ssr/index.md [hunk 18]
```diff
@@ -0,0 +1,111 @@
+# 服务端渲染 (SSR){#server-side-rendering-ssr}
+
+:::tip
+如果你使用的是 **Nuxt.js**，你需要阅读的是[**这些说明文档**](./nuxt.md)。
+:::
+
+只要你在 `setup` 函数、`getters` 和 `actions` 的顶部调用 `useStore()` 函数，使用 Pinia 创建 store 对于 SSR 来说应该是开箱即用。
```
Comment (awxiaoxian2020) [inline packages/docs/zh/ssr/index.md]:
```suggestion
只要你在 `setup` 函数、`getters` 和 `actions` 的顶部调用 `useStore()` 函数，使用 Pinia 创建 store 对于 SSR 来说应该是开箱即用的。
```

File: packages/docs/zh/introduction.md [hunk 19]
```diff
@@ -0,0 +1,192 @@
...
+<VueSchoolLink
+  href="https://vueschool.io/lessons/introduction-to-pinia"
+  title="Get started with Pinia"
+/>
+
+Pinia [起始](https://github.com/vuejs/pinia/commit/06aeef54e2cad66696063c62829dac74e15fd19e)于 2019 年 11 月左右的一场实验，该实验的目的是设计一个拥有 [组合式 API](https://github.com/vuejs/composition-api) 的 Store。从那时起，同时支持 Vue2 和 Vue3 以及不强制要求使用组合式 API 的初心被保留至今。除了**安装**和 **SSR** 等章节之外，其余章节内容的 API 均支持 Vue3 和 Vue2。虽然本文档主要是面向 Vue3，但在必要时会标注出 Vue2 的内容，因此 Vue2 和 Vue3 的用户都可以阅读本文档。
+
+## 为什么你应该使用 Pinia？{#why-should-i-use-pinia}
+
+Pinia 是 Vue 的专属状态管理库，它允许你跨组件/页面共享状态。如果你熟悉组合式 API 的话，你可能会认为你可以通过一行简单的 `export const state = reactive({})`来共享一个全局状态。对于单页应用程序来说确实是这样的，但如果在服务器端渲染，这可能会使您的应用程序暴露出一些安全漏洞。 若是使用 Pinia，即使在小型单页应用程序中，你也可以获得如下功能：
```
Comment (awxiaoxian2020) [inline packages/docs/zh/introduction.md]:
```suggestion
Pinia 是 Vue 的专属状态管理库，它允许你跨组件/页面共享状态。如果你熟悉组合式 API 的话，你可能会认为你可以通过一行简单的 `export const state = reactive({})`来共享一个全局状态。对于单页应用程序来说确实是这样的，但如果在服务器端渲染，这可能会使您的应用程序暴露出一些安全漏洞。 如果使用 Pinia，即使在小型单页应用程序中，你也可以获得如下功能：
```

File: packages/docs/zh/introduction.md [hunk 20]
```diff
@@ -0,0 +1,192 @@
...
+      // 你可以直接 mutate 该状态
+      this.todos.push({ text, id: this.nextId++, isFinished: false })
+    },
+  },
+})
+```
+
+## 对比 Vuex {#comparison-with-vuex}
+
+Pinia 起源于一场探索 Vuex 下一个迭代的实验，所以其结合了 Vuex 5 核心团队讨论中的许多想法。最终，我们意识到 Pinia 已经实现了我们在 Vuex 5 中想要的大部分功能，所以决定将其作为新的推荐方案来代替 Vuex。
```
Comment (awxiaoxian2020) [inline packages/docs/zh/introduction.md]:
```suggestion
Pinia 起源于一次探索 Vuex 下一个迭代的实验，所以其结合了 Vuex 5 核心团队讨论中的许多想法。最终，我们意识到 Pinia 已经实现了我们在 Vuex 5 中想要的大部分功能，所以决定将其作为新的推荐方案来代替 Vuex。
```

File: packages/docs/zh/introduction.md [hunk 21]
```diff
@@ -0,0 +1,192 @@
...
+
+> Vuex 3.x 只适配 Vue2，而 Vuex 4.x 是适配 Vue3 的。
+
+Pinia API 与 Vuex ≤4 有很大不同，即：
+
+- _mutations_ 已弃用。它们经常被认为是**极其冗长的**。它们初衷是带来 devtools 的集成方案，但这已不再是一个问题了。
+- 无需要创建自定义的复杂包装器来支持 TypeScript，一切都被类型化了，API 的设计方式是尽可能地利用 TS 类型推理。
+- 无过多的魔法字符串注入，只需要导入函数并调用它们，然后享受 autocompletion 的乐趣就好。
+- 无需要动态添加 Store，它们默认都是动态的，甚至你都不会注意到这点。注意，你仍然可以在任何时候手动使用一个 Store 来注册它，但因为它是自动的，所以你不需要担心它。
+- 不再有嵌套结构的**模块**。你仍然可以通过导入和使用另一个 Store 来隐含地嵌套 stores 空间，虽然是 Pinia 从设计上提供的是一个扁平的结构，但仍然能够在 Store 之间进行交叉组合。**你甚至可以让 Stores 有循环依赖关系**。
```
Comment (awxiaoxian2020) [inline packages/docs/zh/introduction.md:189]: 英文单词是否一律使用单数较为合适？

**Review by awxiaoxian2020 — CHANGES_REQUESTED** [review]

File: packages/docs/zh/introduction.md (hunk 21 above)
Comment (KimYangOfCat) [reply packages/docs/zh/introduction.md:189]: 这个问题，我并不确定，还是得具体问题具体分析吧，不过后续会参考 [Vue 中文文档翻译须知](https://github.com/vuejs-translations/docs-zh-cn/wiki/%E7%BF%BB%E8%AF%91%E9%A1%BB%E7%9F%A5)中的一些适用要求，统一调整一些格式问题。感谢你的校对，多谢！

File: packages/docs/zh/api/modules/pinia_nuxt.md [hunk 22]
```diff
@@ -4,28 +4,28 @@ editLinks: false
 sidebarDepth: 3
 ---
 
-[API Documentation](../index.md) / @pinia/nuxt
+[API 文档](../index.md) / @pinia/nuxt
 
 # Module: @pinia/nuxt
 
-## Interfaces
+## 接口{#interfaces}
```
Comment (awxiaoxian2020) [inline packages/docs/zh/api/modules/pinia_nuxt.md]:
```suggestion
## 接口 {#interfaces}
```

File: packages/docs/zh/api/modules/pinia_nuxt.md [hunk 23]
```diff
@@ -4,28 +4,28 @@ editLinks: false
...
 
 - [ModuleOptions](../interfaces/pinia_nuxt.ModuleOptions.md)
 
-## Functions
+## 函数{#functions}
 
-### default
+### 默认值{#default}
```
Comment (awxiaoxian2020) [inline packages/docs/zh/api/modules/pinia_nuxt.md]:
```suggestion
### 默认值 {#default}
```

File: packages/docs/zh/api/modules/pinia_nuxt.md [hunk 24]
```diff
@@ -4,28 +4,28 @@ editLinks: false
...
 
 ▸ **default**(`this`, `inlineOptions`, `nuxt`): `void` \| `Promise`<`void`\>
 
-#### Parameters
+#### 参数{#parameters}
```
Comment (awxiaoxian2020) [inline packages/docs/zh/api/modules/pinia_nuxt.md]:
```suggestion
#### 参数 {#parameters}
```

File: packages/docs/zh/api/modules/pinia_nuxt.md [hunk 25]
```diff
@@ -4,28 +4,28 @@ editLinks: false
...
 
 | Name | Type |
 | :------ | :------ |
 | `this` | `void` |
 | `inlineOptions` | [`ModuleOptions`](../interfaces/pinia_nuxt.ModuleOptions.md) |
 | `nuxt` | `Nuxt` |
 
-#### Returns
+#### 返回值{#returns}
```
Comment (awxiaoxian2020) [inline packages/docs/zh/api/modules/pinia_nuxt.md]:
```suggestion
#### 返回值 {#returns}
```

**Review by awxiaoxian2020 — CHANGES_REQUESTED** [review]

**Review by awxiaoxian2020 — APPROVED** [review]
LGTM.

I think it should be merged first and improve later.

**Comment by KimYangOfCat** [comment]
> I think it should be merged first and improve later.
@awxiaoxian2020  Agree! please approve this PR.
@posva  PTAL

**Comment by posva** [comment]
Thanks a lot for this @KimYangOfCat 
I'm still waiting for vitepress to support i18n in their newer versions, that way, the font will also be changed. Hopefull this change should be really close

**Comment by KimYangOfCat** [comment]
@posva 
I've looked at the progress of vitepress i18n and it's only about 50%, so there may be some time before it's actually released. Perhaps you could consider merging this PR first, and then I would upgrade the dependencies and modify the configuration once vitepress i18n is officially released. I believe this would not cause any more work, and it would be a good transition solution.
Do you have any other concerns?

**Review by posva — CHANGES_REQUESTED** [review]
Can you remove the changes to the pnpm-lock? Rebasing should work. I think we can go ahead and merge this. We will migrate later on to the latest vitepress

**Comment by KimYangOfCat** [comment]
pnpm-lock changes removed. thanks for merging! 
In the future, I am willing to help fix any problems about the Chinese documentation.

**Review by posva — APPROVED** [review]
Thanks!

**Comment by arnonrdp** [comment]
Amazing merge!
Does this mean that we now have open doors for other languages?
@KimYangOfCat, can we have some README.md with basic instructions for adding other languages?

**Comment by posva** [comment]
Thanks for your interest! Other languages will be kept in other repositories, following the guide at https://github.com/vuejs-translations/guidelines/blob/main/README.md

**Comment by arnonrdp** [comment]
Right, but they will be in the language dropdown of the main site (https://pinia.vuejs.org/) or it will be another url.

**Comment by posva** [comment]
in the dropdown, pointing to an external site

**Comment by KimYangOfCat** [comment]
Later, will the docs package be split into a new repository of the vuejs-translation org? Same as vue docs?
maybe we should open a new issue to discuss this.

**Comment by awxiaoxian2020** [comment]
> Later, will the docs package be split into a new repository of the vuejs-translation org? Same as vue docs? maybe we should open a new issue to discuss this.

See also https://github.com/vuejs-translations/docs-zh-cn/discussions/561

**Pull request merged**